packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -260,7 +284,12 @@ class ScrollDragController implements Drag {
     // We negate the velocity here because if the touch is moving downwards,
     // the scroll has to move upwards. It's the same reason that update()
     // above negates the delta before applying it to the scroll offset.
-    delegate.goBallistic(-velocity);
+    velocity = -velocity;
```

Comment:
ok instead of flipping the sign half a dozen times let's just change the earlier test to `if (!_reversed)` :-)